src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/ResultsViewExpansion.cs

```diff
@@ -10,17 +10,18 @@ namespace Microsoft.CodeAnalysis.ExpressionEvaluator
 {
     internal sealed class ResultsViewExpansion : Expansion
     {
-        internal static ResultsViewExpansion CreateExpansion(DkmClrValue value, Formatter formatter)
+        internal static ResultsViewExpansion CreateExpansion(DkmInspectionContext inspectionContext, DkmClrValue value, Formatter formatter)
         {
             var enumerableType = GetEnumerableType(value);
             if (enumerableType == null)
             {
                 return null;
             }
-            return CreateExpansion(value, enumerableType, formatter);
+            return CreateExpansion(inspectionContext, value, enumerableType, formatter);
         }
 
-        internal static DkmEvaluationResult CreateResultsOnly(
+        internal static EvalResultDataItem CreateResultsOnly(
```

Comment:
In this case, I think it means "results only expansion", rather than "DkmEvaluationResult", so yes, I think it's still an applicable name.